app/assets/stylesheets/frontend/print/_html-publication.scss

```diff
@@ -0,0 +1,21 @@
+.html-publications-show {
+  nav.in-page-navigation {
+    display: block!important;
```

Comment:
That's ok. Not hard to remove the other !important. I think it would be very time consuming checking all the possible knock on effects as it looks like it could effect a lot of things. I did consider removing the other !important but decided on doing it this way for this quick fix to show the html pubs content list.

This is the what it's overriding -
body #global-user-location, body .related-positioning, header.page-header div h1 span, body .actions, body nav, #local-locator-form, #location-loading, #get-started, .finding_location, .found_location, .loading_location, .location_error, #global-browser-prompt, #user-satisfaction-survey, .report-a-problem-toggle, .report-a-problem-container, .show-all-parts {
display: none !important;
}